src/main/kotlin/no/skatteetaten/aurora/fergus/service/StorageGridService.kt

```diff
@@ -93,17 +97,29 @@ class StorageGridServiceReactive(
         val displayGroupName = createDisplayGroupName(bucketName, path, access)
 
         // Get group by shortname
-        val getGroupResponse = storageGridGroupsApi
-            .orgGroupsGroupShortNameGet(shortGroupName)
-            .awaitSingle()
-        if (getGroupResponse.status === GetPatchPostPutGroupResponse.StatusEnum.ERROR) {
-            throw ResponseStatusException(
-                HttpStatus.INTERNAL_SERVER_ERROR,
-                "The Storagegrid groups api returned an error on orgGroupsGroupShortNameGet"
-            )
+        var getGroupResponse: GetPatchPostPutGroupResponse?
+        try {
+            getGroupResponse = storageGridGroupsApi
+                .orgGroupsGroupShortNameGet(shortGroupName)
+                .awaitSingle()
+            if (getGroupResponse != null && getGroupResponse.status === GetPatchPostPutGroupResponse.StatusEnum.ERROR) {
```

Comment:
Er det behov for referential equality her? (===) Jeg har ikke sett dette så mye brukt i kotlin.